### PR TITLE
forgot to enable MPI in the GUI

### PR DESCRIPTION
**Pull request by taputechnic**

### CODE REVIEW

File: sphire/doc/recons3d_n.txt [hunk 1]
```diff
@@ -15,15 +15,15 @@ Usage in command line
 
 Create a binary mask dilated by 2 pixel using the binary threshold from chimera
 
-  sp_recons3d_n.py input_volume output_directory 
+  mpirun -np 32 sp_recons3d_n.py input_volume output_directory
```
Comment (mstabrin) [inline sphire/doc/recons3d_n.txt]: mpirun -np 32 sp_recons3d_n.py input_volume output_directory **--MPI**

File: sphire/doc/recons3d_n.txt [hunk 2]
```diff
@@ -15,15 +15,15 @@ Usage in command line
 
 Create a binary mask dilated by 2 pixel using the binary threshold from chimera
 
-  sp_recons3d_n.py input_volume output_directory 
+  mpirun -np 32 sp_recons3d_n.py input_volume output_directory 
 
 Create an adapted mask dilated by 2 pixel and a soft edge of 5 pixel using the binary threshold from chimera
 
-  sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1
+  mpirun -np 32 sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1
```
Comment (mstabrin) [inline sphire/doc/recons3d_n.txt]: mpirun -np 32 sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1 **--MPI**

File: sphire/doc/recons3d_n.txt [hunk 3]
```diff
@@ -15,15 +15,15 @@ Usage in command line
...
 
 Create an adapted mask dilated by 2 pixel and a soft edge of 5 pixel using the binary threshold from chimera and masked by a soft edged cylinder to 80% (e.g. for helical reconstruction with a box size of 300)
 
-  sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1 --second_mask_shape=cylinder --s_radius=40 --s_nx=240 --s_ny=240 --s_nz=240 --s_ndilation=1 --s_edge_width=5
+  mpirun -np 32 sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1 --second_mask_shape=cylinder --s_radius=40 --s_nx=240 --s_ny=240 --s_nz=240 --s_ndilation=1 --s_edge_width=5
```
Comment (mstabrin) [inline sphire/doc/recons3d_n.txt]: mpirun -np 32 sp_recons3d_n.py input_volume output_directory --threshold=0.3 --edge_width=5 --ndilation=1 --second_mask_shape=cylinder --s_radius=40 --s_nx=240 --s_ny=240 --s_nz=240 --s_ndilation=1 --s_edge_width=5 **--MPI**

File: sphire/doc/recons3d_n.txt [hunk 4]
```diff
@@ -15,15 +15,15 @@ Usage in command line
```
Comment (mstabrin) [inline sphire/doc/recons3d_n.txt]: sp_recons3d_n.py exists in for both MPI and non-MPI versions.

**Comment by taputechnic** [comment]
Resolved in #411

**Pull request closed without merging**